### PR TITLE
Add package and system info

### PR DESCRIPTION
**Pull request by vilyapilya**

### CODE REVIEW

File: lib/logdna.rb [hunk 1]
```diff
@@ -20,50 +22,25 @@ def initialize(key, opts={})
...
-
-      begin
-        request = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json')
-        request.basic_auth 'username', key
-      rescue => e
-        handle_exception(e)
-        return
-      end
+      request = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json')
+      request['user-agent'] = "ruby/#{LogDNA::VERSION} (#{RUBY_PLATFORM.split('-').last})"
```
Comment (smusali) [inline lib/logdna.rb]: No need for system info

File: lib/logdna.rb [hunk 2]
```diff
@@ -3,8 +3,10 @@
 # require 'singleton'
 require 'socket'
 require 'uri'
+require 'rbconfig'
```
Comment (dchai76) [inline lib/logdna.rb]: Let's move this up to maintain alphabetical order.

File: lib/logdna.rb [hunk 3]
```diff
@@ -20,50 +22,26 @@ def initialize(key, opts={})
...
-      end
-
-      begin
-        request = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json')
-        request.basic_auth 'username', key
-      rescue => e
-        handle_exception(e)
-        return
       end
+      p opts
```
Comment (dchai76) [inline lib/logdna.rb]: Remove

File: lib/logdna.rb [hunk 4]
```diff
@@ -20,50 +22,26 @@ def initialize(key, opts={})
...
+      request = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json')
+      pkg_version = opts[:rails_logdna_v] ? opts[:rails_logdna_v] : "ruby/#{LogDNA::VERSION}"
+      request['user-agent'] = "#{pkg_version}"
```
Comment (dchai76) [inline lib/logdna.rb]: Why not `request['user-agent'] = opts[:rails_logdna_v] || "ruby/#{LogDNA::VERSION}"`?

File: lib/logdna.rb (hunk 3 above)
Comment (vilyapilya) [reply lib/logdna.rb]: I think I accidentally  pushed my debugging branch here, sorry.

**Pull request closed without merging**